rad/src/operators/array.rs

```diff
@@ -229,3 +276,326 @@ fn test_filter_negative() {
+            Value::Integer(MapGet as i128),
+            Value::Text("key2".to_string()),
+        ]),
+        Value::Integer(BytesAsInteger as i128),
+    ];
+    let output = sort(&input, &script).unwrap_err();
+
+    assert_eq!(output.to_string(), "Failed to get key `key2` from RadonMap")
+}
+
+#[test]
+fn test_sort_maps_without_byte_decoder() {
+    use crate::operators::RadonOpCodes::MapGet;
+    use crate::types::{bytes::RadonBytes, map::RadonMap};
+    use std::collections::HashMap;
+    let mut map1 = HashMap::new();
+
+    map1.insert("key1".to_string(), RadonBytes::from(Value::Integer(0)));
+    map1.insert("key2".to_string(), RadonBytes::from(Value::Integer(1)));
+    let input = RadonArray::from(vec![RadonMap::from(map1.clone()).into()]);
+    let script = vec![Value::Array(vec![
+        Value::Integer(MapGet as i128),
+        Value::Text("key2".to_string()),
+    ])];
+    let output = sort(&input, &script).unwrap_err();
+
+    assert_eq!(
+        output.to_string(),
+        "ArraySort is not supported for RadonArray with inner type `RadonBytes`"
+    )
+}
+
+#[test]
+fn test_sort_map_wrong_decode() {
+    use crate::operators::RadonOpCodes::{BytesAsString, MapGet};
+    use crate::types::{bytes::RadonBytes, map::RadonMap};
+    use std::collections::HashMap;
+    let mut map1 = HashMap::new();
+    map1.insert("key1".to_string(), RadonBytes::from(Value::Integer(0)));
+    map1.insert("key2".to_string(), RadonBytes::from(Value::Integer(1)));
+
+    let input = RadonArray::from(vec![RadonMap::from(map1.clone()).into()]);
+    let script = vec![
+        Value::Array(vec![
+            Value::Integer(MapGet as i128),
+            Value::Text("key2".to_string()),
+        ]),
+        Value::Integer(BytesAsString as i128),
+    ];
+    let output = sort(&input, &script).unwrap_err();
+
+    assert_eq!(
+        output.to_string(),
+        "Failed to decode RadonString from serde_cbor::value::Value"
+    )
+}
+
+#[test]
+fn test_sort_map_floats_value() {
+    use crate::operators::RadonOpCodes::{BytesAsFloat, MapGet};
+    use crate::types::{bytes::RadonBytes, map::RadonMap};
+    use std::collections::HashMap;
+    let mut map1 = HashMap::new();
+    map1.insert(
+        "key1".to_string(),
+        RadonBytes::from(Value::Float(std::f64::consts::PI)),
+    );
+    map1.insert(
+        "key2".to_string(),
+        RadonBytes::from(Value::Float(std::f64::consts::PI)),
+    );
+
+    let input = RadonArray::from(vec![RadonMap::from(map1.clone()).into()]);
+    let script = vec![
+        Value::Array(vec![
+            Value::Integer(MapGet as i128),
+            Value::Text("key2".to_string()),
+        ]),
+        Value::Integer(BytesAsFloat as i128),
+    ];
+    let output = sort(&input, &script).unwrap_err();
+
+    assert_eq!(
+        output.to_string(),
+        "ArraySort is not supported for RadonArray with inner type `RadonFloat`"
+    )
+}
+
+#[test]
+fn test_operate_sort_string_2_arrays() {
```

Comment:
```suggestion
fn test_sort_string_2_arrays() {
```

Reply:
I think `operate` is used for tests which directly use the `operate()` function, can you remove it from all this test names?